test/org/sagebionetworks/bridge/services/backfill/AsyncBackfillTemplateTest.java

```diff
@@ -148,7 +151,15 @@ public boolean matches(Object item) {
                 long expireInMillis = TestBackfillService.EXPIRE * 1000L;
                 // Make sure the time point after which we look for the list of backfill tasks
                 // goes back for the duration of lock expiration
-                return beforeBackfill - expireInMillis < since && since < afterBackfill - expireInMillis;
+                long lower = beforeBackfill - expireInMillis;
+                long upper = afterBackfill - expireInMillis;
```

Comment:
We call getTasks() with now - expireInMillis. The test doesn't mock out "now", so we don't know exactly when getTasks will be called.

However, we do keep track of when we started the backfill and when we finished, so we can bound now to between beforeBackfill and afterBackill. So we can bound when getTasks() to between (beforeBackfill - expireInMillis) and (afterBackfill - expireInMillis).